src/main/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheck.java

```diff
@@ -360,7 +360,7 @@ protected void processFiltered(File file, FileText fileText) {
     }
 
     @Override
-    @SuppressWarnings("determinism:argument.type.incompatible")  // true positive; logging: Iteration over OrderNonDet collection
+    @SuppressWarnings("determinism:argument.type.incompatible")  // true positive (Fixed); logging: Iteration over OrderNonDet collection
```

Comment:
If this is fixed, why is the warning still present?  Can you update to the latest Checkstyle or at least cherry-pick the bug fix?